Update ball.v

ysx001 opened the pull request:
Updates on Ball Movement

ysx001 committed:
Update ball.v

twoodford committed:
Merge branch 'master' into ysx001-patch-1

twoodford commented:
A few notes here:
- For RGB outputs, R is 3 bits, G is 3 bits, and B is 2 bits for a total of 8 bits
- It looks like `ball_on` tells you whether the ball is on the board.  Could you clarify that?
- `ball_on` is currently declared as a 3-bit value, when it should probably be 1-bit

Otherwise, this looks like it's close to being ready to try to compile and test it

ysx001 committed:
Update ball.v

loll70 committed:
Update ball.v

Line 19:     not needed as the work logic changed.
Line 50: 	assign ball_on = (x > 30 && x < 600 && y > 0 && y < 480)
Line 64:    if (ball_top <= 3) so that ball will be displayed even when bounced.
Line 66:    same as above

ysx001 committed:
Update ball.v
